pkg/util/util_test.go

```diff
@@ -31,16 +31,16 @@ func init() {
 }
 
 func TestThroughputCalc(t *testing.T) {
```

Comment:
Can't we delete both this test and the util.ThroughputCalc struct? I don't think it's ever used anymore?